### PR TITLE
TG-58 Updated POST/games

### PR DESCRIPTION
**Pull request by besxtd**
Updated the post function in routes. Not 100% sure if it works, will need some testing.

### CODE REVIEW

File: routes/games.js [hunk 1]
```diff
@@ -11,7 +11,26 @@ router.get(path + '/', (req, res) => {
...
+        message: ''
+    }
+    const requiredValues = ['name', 'iconUrl', 'bannerUrl', 'genres', 'platforms']
+    for (const value of requiredValues) {
+        if (typeof reqData[value] === 'undefined') {
+            response.message = 'Required field ' + value + ' is missing or blank'
+            return res.status(400).json(response)
+        }
+    }
+    users.createGame(reqData.name, reqData.iconUrl, reqData.bannerUrl, reqData.genres, reqData.platforms).then(() => {
```
Comment (jpadgett7) [inline routes/games.js]: Shouldn't this be games.createGame?

Comment (besxtd) [reply routes/games.js]: yep

**Comment by besxtd** [comment]
Ok the right word should be there now

**Comment by CastroDan** [comment]
Fixed the issues with the function and fixed indentation. Code should be gucci now, will wait for other comments though

**Review by CastroDan — APPROVED** [review]

**Pull request merged**